Crashlytics/UnitTests/Mocks/FIRCLSMockFileManager.h

```diff
@@ -26,13 +26,25 @@
 // Incremented when a remove happens with removeItemAtPath
 @property(nonatomic) NSInteger removeCount;
 
+// Overrides fileSizeAtPath if set
+@property(nonatomic, copy) NSNumber *fileSizeAtPathResult;
+
 // Will be fulfilled when the expected number of removes have happened
 // using removeItemAtPath
 //
 // Users should initialize this in their test.
 @property(nonatomic, strong) XCTestExpectation *removeExpectation;
 
+@property(nonatomic, copy) NSString *removedItemAtPath_path;
+
 // Overriding the method for testing Settings
 - (BOOL)removeItemAtPath:(NSString *)path;
 
+// Overrides moveItemAtPath if set
+@property(nonatomic) NSNumber *moveItemAtPathResult;
+@property(nonatomic, copy) NSString *moveItemAtPath_path;
+@property(nonatomic, copy) NSString *moveItemAtPath_destDir;
```

Comment:
How do you feel about checking the files directly instead of setting variables?

Reply:
Discussed offline:
I have a concern with making changes to the file system in unit tests. I have those tests cause unreliability to tests. One of the things I don’t like with the mock file manager is that it actually calls to the actual file manager. Mocks should not be calling into the product equivalent of itself. We have a good opportunity to fix these unit tests when we remove the experiment and perform a larger refactor.